tools/nd_freeze_install

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/python3
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Script to install older packages based on version number.
+#
+# If running this in a stripped down environment such as a docker container,
+# install python dependencies with this command:
+#
+# apt-get update && apt-get install -y python3 python3-pip && pip3 install requests bs4
```

Comment:
this one is for debian systems, thus best to avoid pip installations... should use debian packages, thus better recommend just `apt-get install -t python3-requests python3-bs4`.  Obviously some VERY older systems might not have them but in general they seems to be available: https://packages.debian.org/search?suite=all&searchon=names&keywords=python3-bs4  https://packages.debian.org/search?suite=all&section=all&arch=any&searchon=names&keywords=python3-requests